string type
float64 x_obj
float64 y_obj
float64 x_wall1
float64 y_wall1
float64 x_wall2
float64 y_wall2